aspnet/fundamentals/dependency-injection.rst

```diff
@@ -51,51 +50,47 @@ Registering Your Own Services
 
 In the default web template example above, two application services are added to the ``IServiceCollection``. 
 
-.. literalinclude:: ../../common/samples/WebApplication1/src/WebApplication1/Startup.cs
+.. literalinclude:: dependency-injection/sample/src/DependencyInjectionSample/Startup.cs
   :language: c#
-  :linenos:
-  :lines: 53-55
+  :lines: 52-54
   :dedent: 12
-  :emphasize-lines: 2-3
 
-.. note:: Keep in mind, in addition to these two types, everything else in ``ConfigureServices`` is about adding services. For example, ``services.AddMvc()`` adds the services MVC requires.
+.. note:: Keep in mind, in addition to these two types, everything else in ``ConfigureServices`` is also adding and configuring services. For example, ``services.AddMvc()`` adds the services MVC requires.
```

Comment:
I don't think you need this note. IF you want to keep it, lose all the dead wood.
Each ``services.Add<service>`` call adds (and potentially configures) services .For example, ``services.AddMvc`` add MVC.